### ISSUE TITLE
Breaking 

### ISSUE THREAD

**Issue by NicholasKao1029** [Oneflow-Inc/onediff#199]
```python
import oneflow as flow

flow.mock_torch.enable()
from onediff import OneFlowStableDiffusionPipeline
from diffusers.schedulers import (
    DPMSolverMultistepScheduler,
)
model_id = "runwayml/stable-diffusion-v1-5"
schedulers = {
    "DPM++": DPMSolverMultistepScheduler.from_pretrained(model_id, subfolder="scheduler"),
}

pipe = OneFlowStableDiffusionPipeline.from_pretrained(
    model_id,
    use_auth_token=True,
    torch_dtype=flow.float16,
)

import time 
from collections import defaultdict

pipe = pipe.to("cuda")
pipe.set_graph_compile_cache_size(9)
pipe.enable_graph_share_mem()
pipe.enable_save_graph()
pipe.scheduler = schedulers["DPM++"]
print("here")

images = pipe(
    prompt="""
    A horse
    """,
    negative_prompt="""
    ugly
    """
    ,
    height=768,
    width=512,
    num_inference_steps=60,
    guidance_scale=7,
    output_type="np",
).images
print('finished')
for j, image in enumerate(images):
    print("saving")
    image.save(f"./output/{i}.png")
    print("finsihed saving")
```

Breaks with updated oneflow I believe

```
root@2b9cf9e80ef6:/workspace/oneflow-server# python example/blah.py
Fetching 15 files: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 15/15 [00:00<00:00, 230456.26it/s]
Cannot initialize model with low cpu memory usage because `accelerate` was not found in the environment. Defaulting to `low_cpu_mem_usage=False`. It is strongly recommended to install `accelerate` for faster and less memory-intense model loading. You can do so with:
```
pip install accelerate
```
.
`text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["id2label"]` will be overriden.
/opt/conda/lib/python3.8/site-packages/transformers/models/clip/feature_extraction_clip.py:28: FutureWarning: The class CLIPFeatureExtractor is deprecated and will be removed in version 5 of Transformers. Please use CLIPImageProcessor instead.
  warnings.warn(
here
W20230418 21:50:39.682742  8409 cuda_stream.cpp:48] Runtime version 8.4 of cuDNN incompatible with compiletime version 8.5.
  0%|                                                                                                                                                                                                                                  | 0/60 [00:00<?, ?it/s]F20230418 21:50:52.901175  8531 cutlass_conv_tuner.cpp:278] Check failed: cudaStreamSynchronize(benchmark_stream) : an illegal memory access was encountered (700)
*** Check failure stack trace: ***
    @     0x7f90d79de8aa  google::LogMessage::Fail()
    @     0x7f90d79e17e1  google::LogMessage::SendToLog()
    @     0x7f90d79de3d9  google::LogMessage::Flush()
    @     0x7f90d79e20c9  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f90d11e926f  oneflow::CutlassConvTuner::Impl::FindConv2dOperation()
    @     0x7f90d1152dcb  oneflow::(anonymous namespace)::Conv2dCutlassKernel::Compute()
    @     0x7f90d04480fd  oneflow::UserKernel::ForwardUserKernel()
    @     0x7f90d04482cb  oneflow::UserKernel::ForwardDataContent()
    @     0x7f90d0414433  oneflow::Kernel::Forward()
    @     0x7f90d0414539  oneflow::Kernel::Launch()
    @     0x7f90d0667ae8  oneflow::(anonymous namespace)::LightActor<>::ProcessMsg()
    @     0x7f90d0c6dccf  oneflow::Thread::PollMsgChannel()
    @     0x7f90d0c6e361  _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7oneflow6ThreadC4ERKNS3_8StreamIdEEUlvE_EEEEE6_M_runEv
    @     0x7f90d79f3bc0  execute_native_thread_routine
    @     0x7f91cbd6f609  start_thread
    @     0x7f91cbb2e133  clone
Stack trace (most recent call last) in thread 8531:
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d79f3bbf, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d0c6e360, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d0c6dcce, in oneflow::Thread::PollMsgChannel()
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d0667ae7, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d0414538, in oneflow::Kernel::Launch(oneflow::KernelContext*) const
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d0414432, in oneflow::Kernel::Forward(oneflow::KernelContext*) const
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d04482ca, in oneflow::UserKernel::ForwardDataContent(oneflow::KernelContext*) const
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d04480fc, in oneflow::UserKernel::ForwardUserKernel(std::function<oneflow::Blob* (std::string const&)> const&, oneflow::user_op::OpKernelState*) const
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d1152dca, in oneflow::(anonymous namespace)::Conv2dCutlassKernel::Compute(oneflow::user_op::KernelComputeContext*, oneflow::user_op::OpKernelState*, oneflow::user_op::OpKernelCache const*) const
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d11e926e, in oneflow::CutlassConvTuner::Impl::FindConv2dOperation(oneflow::ep::CudaStream*, cutlass::library::ConvFunctionalKey, cutlass::library::Conv2dConfiguration const&, cutlass::library::ConvArguments const&, void*, unsigned long)
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d79e20c8, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d79de3d8, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d79e17e0, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90d79de8a9, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f90ca8201ba, in

Aborted (Signal sent by tkill() 8304 0)
Aborted (core dumped)
```

Is there a last known working version?

doctor
```
path: ['/opt/conda/lib/python3.8/site-packages/oneflow']
version: 0.9.1.dev20230417+cu117
git_commit: 15c75402
cmake_build_type: Release
rdma: True
mlir: True
```

**Comment by NicholasKao1029** [comment]
ran `python3 -m onediff.demo` also same or simliar issue
```
.
`text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["id2label"]` will be overriden.
/opt/conda/lib/python3.8/site-packages/transformers/models/clip/feature_extraction_clip.py:28: FutureWarning: The class CLIPFeatureExtractor is deprecated and will be removed in version 5 of Transformers. Please use CLIPImageProcessor instead.
  warnings.warn(
W20230419 03:16:24.894809  4287 cuda_stream.cpp:48] Runtime version 8.4 of cuDNN incompatible with compiletime version 8.5.
F20230419 03:16:37.092123  4182 cutlass_conv_tuner.cpp:283] Check failed: cudaEventSynchronize(end) : an illegal memory access was encountered (700)
*** Check failure stack trace: ***
    @     0x7f499325d8aa  google::LogMessage::Fail()
    @     0x7f49932607e1  google::LogMessage::SendToLog()
    @     0x7f499325d3d9  google::LogMessage::Flush()
    @     0x7f49932610c9  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f498ca681b9  oneflow::CutlassConvTuner::Impl::FindConv2dOperation()
    @     0x7f498bb31512  _ZZNK7oneflow12_GLOBAL__N_127CutlassConvTuningWarmupPass5ApplyEPNS_3JobEPNS_10JobPassCtxEENKUlPKNS_6OpNodeEE_clES8_
    @     0x7f498bb32ade  oneflow::(anonymous namespace)::CutlassConvTuningWarmupPass::Apply()
    @     0x7f498b9d7cc4  _ZZN7oneflow23LazyJobBuildAndInferCtx8CompleteEvENKUlRKSsiE2_clES2_i
    @     0x7f498b9dd040  oneflow::LazyJobBuildAndInferCtx::Complete()
    @     0x7f4a7ee1ba3e  oneflow::CurJobBuildAndInferCtx_Complete()
    @     0x7f4a7ee1f94d  (unknown)
    @     0x7f4a7eb7ad0a  (unknown)
    @     0x5654b59a8e7e  PyCFunction_Call
    @     0x5654b5991631  _PyObject_MakeTpCall
    @     0x5654b598d481  _PyEval_EvalFrameDefault
    @     0x5654b5987461  _PyEval_EvalCodeWithName
    @     0x5654b599933c  _PyFunction_Vectorcall
    @     0x5654b59a8a3b  method_vectorcall
    @     0x5654b59ab172  PyObject_Call
    @     0x5654b598a870  _PyEval_EvalFrameDefault
    @     0x5654b5987461  _PyEval_EvalCodeWithName
    @     0x5654b599933c  _PyFunction_Vectorcall
    @     0x5654b59a8a3b  method_vectorcall
    @     0x5654b59ab172  PyObject_Call
    @     0x5654b598a870  _PyEval_EvalFrameDefault
    @     0x5654b5987461  _PyEval_EvalCodeWithName
    @     0x5654b599933c  _PyFunction_Vectorcall
    @     0x5654b59a8a3b  method_vectorcall
    @     0x5654b59ab172  PyObject_Call
    @     0x5654b598a870  _PyEval_EvalFrameDefault
    @     0x5654b5987461  _PyEval_EvalCodeWithName
    @     0x5654b599933c  _PyFunction_Vectorcall
Stack trace (most recent call last):
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/_oneflow_internal.cpython-38-x86_64-linux-gnu.so", at 0x7f4a7eb7ad09, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/_oneflow_internal.cpython-38-x86_64-linux-gnu.so", at 0x7f4a7ee1f94c, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/_oneflow_internal.cpython-38-x86_64-linux-gnu.so", at 0x7f4a7ee1ba3d, in oneflow::CurJobBuildAndInferCtx_Complete()
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f498b9dd03f, in oneflow::LazyJobBuildAndInferCtx::Complete()
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f498b9d7cc3, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f498bb32add, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f498bb31511, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f498ca681b8, in oneflow::CutlassConvTuner::Impl::FindConv2dOperation(oneflow::ep::CudaStream*, cutlass::library::ConvFunctionalKey, cutlass::library::Conv2dConfiguration const&, cutlass::library::ConvArguments const&, void*, unsigned long)
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f49932610c8, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f499325d3d8, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f49932607e0, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f499325d8a9, in
   Object "/opt/conda/lib/python3.8/site-packages/oneflow/../oneflow.libs/liboneflow-42b8eae0.so", at 0x7f498609f1ba, in
```

**Comment by jackalcooper** [comment]
we will look into it

**Comment by NicholasKao1029** [comment]
These are the install command I will use is this good?

```
python3 -m pip install --pre oneflow -f https://staging.oneflow.info/branch/master/cu117
python3 -m pip install "torch<2" "transformers>=4.26" "diffusers[torch]==0.14.0"
python3 -m pip uninstall accelerate -y
python3 -m pip install onediff==0.9.0
```

**Comment by NicholasKao1029** [comment]
Nice that worked!

**Comment by jackalcooper** [comment]
> These are the install command I will use is this good?
> 
> ```
> python3 -m pip install --pre oneflow -f https://staging.oneflow.info/branch/master/cu117
> python3 -m pip install "torch<2" "transformers>=4.26" "diffusers[torch]==0.14.0"
> python3 -m pip uninstall accelerate -y
> python3 -m pip install -U onediff
> ```

this seems to be OK. I'm using the same commands

**Comment by NicholasKao1029** [comment]
> 0.9.1.dev20230419+cu117

How can I download this specific commit, it seems the latest nightly has broken something for me

**Comment by jackalcooper** [comment]
> > 0.9.1.dev20230419+cu117
> 
> Hi @jackalcooper this worked great, but the new nightly has broken how can I download this specific commit?

In principle, we would recommend you stay updated with latest nightly. Unfortunately we don't have the manpower and bandwidth to maintain old nightly versions.

**Comment by NicholasKao1029** [comment]
Old nightly builds are just completely removed then? if so then I'll wait or try downgrading one-diff and oneflow in the meantime

**Comment by jackalcooper** [comment]
> Old nightly builds are just completely removed then? if so then I'll wait or try downgrading one-diff and oneflow in the meantime

as or now latest version of onediff is supposed to work with latest version of oneflow nightly. in the future there will be a onediff version with a fixed version of release version of oneflow (0.10.0 maybe).
